config/scorecard/kustomization.yaml

```diff
@@ -3,17 +3,18 @@
 
 resources:
 - bases/config.yaml
-patchesJson6902:
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+patches:
 - path: patches/basic.config.yaml
   target:
     group: scorecard.operatorframework.io
-    version: v1alpha3
     kind: Configuration
     name: config
+    version: v1alpha3
 - path: patches/olm.config.yaml
   target:
     group: scorecard.operatorframework.io
-    version: v1alpha3
     kind: Configuration
     name: config
-#+kubebuilder:scaffold:patchesJson6902
```

Comment:
Do we still need the kubebuilder marker?

Reply:
Good catch,  both this one and the other one you mentioned were auto-removed by `kustomize edit fix` so I assumed it was correct, let me dig into it on monday!